src/test/resources/pacts/publicapi-connector-get-payment-refunds.json

```diff
@@ -0,0 +1,210 @@
+                    "href": "https://app.com/v1/api/accounts/123456/charges/charge8133029783750222"
+                  }
+                }
+              },
+              {
+                "amount": 1,
+                "created_date": "2016-01-25T16:23:55.000Z",
+                "refund_id": "m16ufgc3t23l766ljhv9eicsn5",
+                "user_external_id": "d77458fb878b4aba88b8164368be3d16",
+                "status": "error",
+                "_links": {
+                  "self": {
+                    "href": "https://app.com/v1/api/accounts/123456/charges/charge8133029783750222/refunds/m16ufgc3t23l766ljhv9eicsn5"
+                  },
+                  "payment": {
+                    "href": "https://app.com/v1/api/accounts/123456/charges/charge8133029783750222"
+                  }
+                }
+              }
+            ]
+          }
+        },
+        "matchingRules": {
+          "body": {
+            "$.payment_id": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$._links.self.href": {
+              "matchers": [
+                {
+                  "regex": "http.*://.*/v1/api/accounts/123456/charges/charge8133029783750222/refunds"
+                }
+              ]
+            },
+            "$._links.payment.href": {
+              "matchers": [
+                {
+                  "regex": "http.*://.*/v1/api/accounts/123456/charges/charge8133029783750222"
+                }
+              ]
+            },
+            "$._embedded.refunds[0].amount": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$._embedded.refunds[0].created_date": {
+              "matchers": [
+                {
+                  "match": "value"
+                }
+              ]
+            },
+            "$._embedded.refunds[0].refund_id": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$._embedded.refunds[0].user_external_id": {
+              "matchers": [
+                {
+                  "match": "type"
+                }
+              ]
+            },
+            "$._embedded.refunds[0].status": {
+              "matchers": [
+                {
+                  "match": "value"
+                }
+              ]
+            },
+            "$._embedded.refunds[0]._links.self.href": {
+              "matchers": [
+                {
+                  "regex": "http.*://.*/v1/api/accounts/123456/charges\/[a-z0-9]*\/refunds\/[a-z0-9]*"
+                }
+              ]
+            },
+            "$._embedded.refunds[0]._links.payment.href": {
+              "matchers": [
+                {
+                  "regex": "http.*://.*/v1/api/accounts/123456/charges/charge8133029783750222"
```

Comment:
Do we want to check whether it's the exact value or rather match it with `\/[a-z0-9]*\`? (same for account id)

Reply:
Values should be fine here as they won't change